README.md

```diff
@@ -114,6 +114,8 @@ Add an environment variable `RSTUDIO_PANDOC` in the .bash_profile that
 points to tne pandoc directory. For example:
 `/Applications/RStudio.app/Contents/MacOS/pandoc`
 
+Alternatively, use install `pandoc` in your system using  the command `conda install pandoc`
```

Comment:
Can you also add this in the README.Rmd too? As usually we generate README.md from the Rmd (as the single source of truth).

Reply:
done